manifests/07-operator.yaml

```diff
@@ -4,7 +4,7 @@ kind: Deployment
 metadata:
   name: cluster-image-registry-operator
   namespace: openshift-image-registry
-  annotations:
+  labels:
```

Comment:
this isn't what changed.  the configmap changed from annotation to label.  The env vars are still injected via annotation on the deployment.